### ISSUE TITLE
2019 08 16 Transportation page wheel icon collision

### ISSUE THREAD

**Issue by taylor-iterators** [CityOfBoston/Iterators#39]
Screengrab/videos (insert file names here)
2019 08 16 Transportation page wheel icon collision.mp4

Link
http://bostond8stg.prod.acquia-sites.com/departments/transportation

Type of User:  Anonymous
 Environment
(Device > OS > Browser)
Mac Desktop > OS 12.14.6 > Safari 12.1.2

Section
(Page > Area > Element)
Transportation main page > Body > Wheel icon

Brief Description
On the Transportation page, there is a centered cold “TRANSPORTATION” title, to the right of which is a contact info column. Atop the column is a red-underlined encircled wheel icon; when the screen width is narrowed, eventually it collides with the page title rather dramatically.
 Steps to reproduce
1.	Go to the Transportation page
2.	Grab on side of the window and narrow it significantly
Desired Result
The page remains orderly.

Actual Result Two very prominent aesthetic elements of the page overlaps a great deal.

[2019 08 16 Transportation page wheel icon collision.zip](https://github.com/CityOfBoston/Iterators/files/3510824/2019.08.16.Transportation.page.wheel.icon.collision.zip)

**Comment by davidrkupton** [comment]
Done.
Sized department icon to 100px square on all department profile pages.

**Comment by taylor-iterators** [comment]
I'm still experiencing the issue; it's perhaps less prominent now but narrowing the window still causes noteworthy overlap. Reassigning this one.

**Comment by davidrkupton** [comment]
Sorry, thought this was an issue with the icon being larger than the circle it is contained within.
This has now been addressed somewhat.  
This issue exists on the current D7 site.
However, I have improved the situation somewhat by setting the icon to move to be right aligned at 1300px (one of our breakpoints) until the general format breaks into a column at 980px (another of our breakpoints).
The issue is obviously dependent on the length of the word in the title, if it is longer than "transportation" there remains potential for worse overlap, but without redesigning (dynamic resizing of font or hiding the icon completely on collision) this is a pragmatic solution.
